docs/assisting-with-translations.md

```diff
@@ -1,20 +1,17 @@
-## Translations
+# Assisting with Translations
 
-### How To Translate
+This document provides information regarding translations for bitcoin.org.
 
-You can join a translation team on
-[Transifex](https://www.transifex.com/projects/p/bitcoinorg/) and start
-translating or improving existing translations.
+## Table of Contents
+* [Handling Translations on GitHub](#handling-translations-on-github)
+* [Getting started with the Translation Team](#getting-started-with-the-translation-team)
```

Comment:
Please capitalize "started" so it matches the capitalization style found in the other headings:

"Getting Started with the Translation Team"

Reply:
Fixed